ResourcesCard.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = (resourceUuid) => {
```

Comment:
This variable is never reassigned, so we can use `const`:

```suggestion
  const sendEventTracking = (resourceUuid) => {
```